app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewsController < ApplicationController
+  def new
+    @current_shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:shelter_id])
+    review = shelter.reviews.create(review_params)
+    redirect_to "/shelters/#{shelter.id}"
```

Comment:
Making note of this because it seems to be working without needing review.save. Interesting.